### PR TITLE
Ft basic chat 165412873

### PR DESCRIPTION
**Pull request by luc-tuyishime**
#### What does this PR do?
- Allow one and multiple users to be able to chat on authors haven application.

#### Description of Task to be completed?
- Design a chat page where users will send and receive messages instantaneously

#### How should this be manually tested?
- After cloning the repo, cd into it and type `npm test`.
- After cloning the repo, cd into it and type `npm start` and follow the instruction to open it in the browser.

#### Any background context you want to provide?
- To be able to send a message you have to create an account start chatting

#### What are the relevant pivotal tracker stories?
 [#165412859](https://www.pivotaltracker.com/story/show/165412859)

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by rwajon — APPROVED** [review]

**Comment by kagaramag** [comment]
Change the PR name to the convenient naming convention

File: package.json [hunk 1]
```diff
@@ -47,23 +47,24 @@
     "react-autosize-textarea": "^7.0.0",
     "react-dom": "^16.8.6",
     "react-draft-wysiwyg": "^1.13.2",
-    "react-html-parser": "^2.0.2",
```
Comment (kagaramag) [inline package.json]: Why did you delete these packages?

File: public/index.html [hunk 2]
```diff
@@ -6,8 +6,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link
```
Comment (kagaramag) [inline public/index.html:9]: There is no need to include this file in `index.html`. We  already have `@fortawesome` and they are serving the same work. This means every page is loading this `font-awesome` regardless if that page needs it or not!

File: src/assets/css/_colors.scss [hunk 3]
```diff
@@ -50,12 +52,16 @@ $twitter-color: #50abf1;
 .light-grey {
   background: $light-grey;
 }
-.light-red {
-  background: $light-red;
+
+.darkenBlue {
+  background-color: $darkenBlue;
 }
-.text-light-grey {
```
Comment (kagaramag) [inline src/assets/css/_colors.scss:56]: Why are you deleting classes that we are using man? introduce new ones but dot delete them.

File: src/components/Chat/Chat.scss [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+$height: 60px;
+$bottom: 40px;
+$right: 40px;
+$bottom-btn: 16px;
+$right-btn: 33px;
+$font-size: 13px;
+$font-size-floating-chat: 25px;
+
+.Chat {
+  .my-float {
```
Comment (kagaramag) [inline src/components/Chat/Chat.scss:18]: CSS has a similar built-in `float`, it can be good if you rename this class in another way

**Review by kagaramag — CHANGES_REQUESTED** [review]

File: src/assets/css/_colors.scss [hunk 5]
```diff
@@ -10,6 +10,8 @@ $info: #1cbdee;
 $light-grey: #cbcbcb;
 $light-red: #f5eeee;
 $yellow: #f9d342;
+$darkenBlue: #1e88e5;
+$greenAccent: #eeeeee;
 /* background anim colors */
```
Comment (kagaramag) [inline src/assets/css/_colors.scss]: - Remove comment
- CSS uses `hyphen-case` not `camelCase`

File: src/components/Chat/customStyles.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+export const customStyles = {
```
Comment (kagaramag) [inline src/components/Chat/customStyles.js]: - As I can see there are only CSS, why cant you put them in you SCSS file?
- If I get it well, this is to style the popup chat modal, `the modal` is always `60%` of the screen, cant it be somehow bad for the small screen. the remaining 40% is a huge space

**Review by kagaramag — CHANGES_REQUESTED** [review]

File: src/assets/css/_colors.scss (hunk 5 above)
Comment (luc-tuyishime) [reply src/assets/css/_colors.scss]: Changes applied

File: src/components/Chat/customStyles.js (hunk 6 above)
Comment (luc-tuyishime) [reply src/components/Chat/customStyles.js]: Changes applied

**Review by kagaramag — APPROVED** [review]
